packages/svelte-ux/src/routes/docs/components/Table/+page.svelte

```diff
@@ -201,6 +203,51 @@
   />
 </Preview>
 
+<h2>Order + Pagination</h2>
+
+<Preview>
+  <Paginate items={sorted_data} perPage={5} let:pageItems let:pagination>
```

Comment:
It might make the example easier to see if the sorting was done in the markup, instead of a separate variable.

```svelte
<Paginate items={data.sort($order.handler)} perPage={5} let:pageItems let:pagination>
```